webapp/src/components/testing/PredictSteps.vue

```diff
@@ -0,0 +1,342 @@
+<template>
+  <!-- Language model prompting is currently unavailable   -->
+  <div
+    v-if="task.trainingInformation.dataType === 'text'"
+    v-show="validationStore.step !== 0"
+  >
+    <div class="flex justify-center items-center mb-4">
+      <span class="shrink-0 py-4 px-4 bg-orange-100 rounded-md">
+        <p class="text-slate-600 text-xs">
```

Comment:
```suggestion
      <span class="shrink-0 py-4 px-4 bg-purple-100 rounded-md
        flex flex-row gap-x-4 items-center"
      >
        <InfoIcon custom-class="min-w-6 min-h-6 w-6 h-6 text-slate-600" />
```
To follow the new purple banner style
Make sure to import InfoIcon :)
```
import InfoIcon from "@/assets/svg/InfoIcon.vue";
```

Reply:
hmm it seems the code suggestion isn't pointing at the right line... it should point to l.8 not l.9